### DISCUSSION TITLE
Bootstrap 5 styles support for small (-sm) select / inputs ?

### DISCUSSION THREAD

**Discussion by martinbarilik** [orchidjs/tom-select#90] [Ideas]
Hi, i just got into tom-select as i was searching for select2 alternative which is not depending on dead-end jquery library. I am working with rails 6 project with bootsrap 5 and everything looks great so far but i am missing bootstrap 5 scss support for small inputs / selects documented in here 

https://getbootstrap.com/docs/5.0/forms/select/#sizing

What do you say ? It's crucial as i am rewriting old rails 3 project and it uses only small selects for more space.

**Comment by oyejorge** [comment]
[I say yes!](https://github.com/orchidjs/tom-select/commit/6ac11ce29369b473aef2a80941574a7d2f3b9d95#diff-6fcebb391c0c0951d998e4bcbac8a903fde9b3fd1cbe2f5c4119e92c81a7f11f)

New release with those changes coming very soon

**Reply by martinbarilik** [reply]
Thank you very much.

https://jsfiddle.net/mArtinko5MB/cx8203az/17/

the small tom-select is pixel or two higher i think, look at the fiddle
